_plugins/joiner.rb

```diff
@@ -127,27 +119,13 @@ def initialize(site)
       create_team_by_email_index
     end
 
-    # Takes Hash<string, Array<Hash>> collections and flattens them
-    # Array<Hash>.
-    def self.flatten_index(index)
-      private_data = index['private']
-      index['private'] = {'private' => private_data.values} if private_data
-      index.values
-    end
-
-    # Joins team member data, converts site.data[team] to a hash of
-    # username => team_member, and assigns team member images.
-    def join_team_data
-      promote_private_data 'team'
-      assign_team_member_images
-    end
-
-    # Joins public and private project data.
-    def join_project_data
-      promote_private_data 'projects'
-
-      if @public_mode
-        @data['projects'].delete_if {|p| p['status'] == 'Hold'}
+    # Populates details of team members in location data
+    def extend_location_data
```

Comment:
@mbland:  If we add full location cross referencing support to the `team_api` gem, would that obviate the need for this method here?